.tool-versions

```diff
@@ -0,0 +1 @@
+golang 1.11
```

Comment:
I'm not terribly opposed to this. But isn't it a better practice to add this to your global `~/.gitignore` file so that you don't impose the tooling on everyone using the repo?

Reply:
I'm not super attached to keeping this, but I've never gotten this argument, we've built this for go 1.11 and you should probably be using that, if you want to use a different one - change the file?

Reply:
I will be using go 1.11 as opposed to 1.10, but that's irrelevant. A few of us aren't using `asdf` at all, so regardless of the go version, this means nothing to us. If everyone committed their tools' dot files, it would bloat the top level folder even more. But if the team agrees to impose `asdf` it's less of an issue.

I do think it can be avoided altogether if your local version management is locally gitignored via `~/.gitignore`. Another somewhat related example is `.nvmrc` which we've already added implying we're all using `nvm`. Although that probably isn't actually true.

Again, I don't feel strongly about this. Just laying out the argument.